guides/tutorials/codelabs/azure-vmss-source-to-prod/index.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: single
+title:  "Azure VM Scale Sets: Source to Prod"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab, you will create a source-to-prod continuous delivery flow from source to production for a simple java based application.
+
+After completing this lab the following workflow will happen: 
+  1. Some code is checked in on gthub.
+  2. Jenkins automatically runs a job to create a new build. 
+  3. Spinnaker deploys the new version of the application to a Test server group and disable the previous deployment.
+  4. The deployment is verified
+  5. Spinnaker redeploys the same image to the production environment and disable the previous deployment.
+  
+This codelab will walk through the steps to  implement this scenario: 
+
+  * Setup the environment and the necessary pre-requisites
+  * Configure Jenkins and Spinnaker
+  * Create the pipeline for the deployment
+
+## 0. Environment pre-requisites 
+
+### Deploy Spinnaker and Jenkins on Azure 
+
+You need the following resources to perform the codelab: 
+- **Azure subscription**: You can create one for free using the [Azure Free trial](https://azure.microsoft.com/free/).
+- **Spinnaker and Jenkins VM**: You can deploy them on your Azure subscription with the [Continuous Deployment quick start template](https://github.com/Azure/azure-quickstart-templates/tree/master/301-jenkins-aptly-spinnaker-vmss) 
+
+From a shell run the following command that will walk you through all the steps necessary to deploy Jenkins and Spinnaker in Azure:

```

Comment:
This could be confusing, I prefer to keep the focus on vmss with a specific note about the option.